src/libraries/Common/src/System/HexConverter.cs

```diff
@@ -275,13 +275,13 @@ public static bool TryDecodeFromUtf16(ReadOnlySpan<char> chars, Span<byte> bytes
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
         public static int FromChar(int c)
         {
-            return c >= CharToHexLookup.Length ? 0xFF : CharToHexLookup[c];
+            return (uint)c >= CharToHexLookup.Length ? 0xFF : CharToHexLookup[c];
         }
 
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
         public static int FromUpperChar(int c)
         {
-            return c > 71 ? 0xFF : CharToHexLookup[c];
```

Comment:
This looks like an off-by-one error in the original code since 'F' is 70.

Reply:
It's not an actual error because `c` being 70 will return 0xFF anyway (from the lookup)